addons/website/static/src/scss/website.scss

```
@@ -2357,6 +2357,11 @@ $ribbon-padding: 100px;
             color: inherit;
         }
     }
+    a {
```

May be you want to select direct child only.
```suggestion
    > a {
```

---

Good point, but not necessary in this case both selectors are functionally identical here. Our current structure has <a> as the only direct child of .o_search_result_item, so .o_search_result_item a and .o_search_result_item > a return the same results. The descendant selector provides flexibility if the markup changes while maintaining the same performance for this use case.